### ISSUE TITLE
The code has many error! and why you use sigmoid_cross_entropy_with_logits in RNN_model_bi?

### ISSUE THREAD

**Issue by Louis-udm** [younggns/comparative-abusive-lang#1]
The code has a lot of errors, and it took me a long time to make it work.
Another problem, the data set is a mutually exclusive 4 classification task, should use softmax_cross_entropy_with_logits, why do you use sigmoid_cross_entropy_with_logits?

**Comment by david-yoon** [comment]
Hi, 
Sorry for making you spent enormous time to fix the code. There was miscommunication between the authors, so the code was not properly merged to this repository. 

Recently, I noticed your comment and modified the codes to the repository (RNN-LTC, RNN-attn, RNN-ctx (context)). You may pull the repository and use the codes. (python 3)
I also put the script to train the model for your reference.
( ./model/reference_script_train.sh )

As you pointed out, the use of softmax cross_entropy function is proper since we target mutually exclusive classification. I tested sigmoid_cross_entropy for experimental purpose. You can also notice the commented-out softmax version. Now I reverted the code to use softmax_cross_entropy.

Personally, I do not have the code to preprocess the dataset. So if you need the preprocessed dataset, please reach me via an email.

Thank you,
David

**Comment by Louis-udm** [comment]
Thanks, I will test your new code for this week.
